packages/wpcom-checkout/src/payment-methods/ideal.tsx

```diff
@@ -3,6 +3,7 @@ import styled from '@emotion/styled';
 import { useSelect, useDispatch, registerStore } from '@wordpress/data';
 import { sprintf } from '@wordpress/i18n';
 import { useI18n } from '@wordpress/react-i18n';
+import classNames from 'classnames';
```

Comment:
Unfortunately we can't use the `classnames` package in the `wpcom-checkout` package without adding it to the dependency array in the package.json file. I'm not sure it's worth adding a whole package dependency just for this purpose, though.

How about [joinClasses](https://github.com/Automattic/wp-calypso/blob/0d203c15168af5266cc036f284cba74211a4dbf0/packages/wpcom-checkout/src/join-classes.ts#L1)? 

```js
import { joinClasses } from '../join-classes';
// ...
const classes = joinClasses(['form-select', isError ? 'is-error' : undefined ]);
```

Reply:
Removed this import, instead we opted to add a styled component and provide the form styling inline in this case.